src/nplinker/pairedomics/podp_antismash_downloader.py

```diff
@@ -28,41 +34,86 @@
 
```

Comment:
```suggestion
GENOME_STATUS_FILENAME="genome_status.txt"
```
Provide an constant variable for this filename, please update the places where this name is used.